src/pages/profile.tsx

```diff
@@ -78,7 +94,84 @@ export default function Page() {
       <Input background="white" 
         value={newCity} onChange={e => setNewCity(e.target.value)} />
     </FormControl>           
+
+    <Divider margin={componentSpacing} />
+
+    <InterviewPreference />
+
     <Button onClick={handleSubmit} variant="brand">保存</Button>
     {isLoading && <Loader loadingText='保存中...'/>}
   </VStack>;
 }
+
+function InterviewPreference({
+
+} : {
+  
+}) {
+  const [date, setDate] = useState(oneMonthDate);
+  const [limit, setLimit] = useState<Limit | undefined>(undefined);
+
+  return <>
+    <FormControl>
+      <FormLabel>面试官偏好</FormLabel>
+      <Checkbox sx={{ '.chakra-checkbox__control': { bg: 'white' } }}>
+        我不是导师，但可以帮助面试学生。
+      </Checkbox>
+    </FormControl>
+
+    <FormControl>
+      <Flex wrap="wrap" alignItems="center">
+        <Checkbox 
+          isChecked={limit !== undefined}
+          onChange={() => setLimit(limit !== undefined ? 
+            undefined : 
+            { noMoreThan: 0,
+              until: date.toDateString()
+            }
+          )}
+          sx={{ '.chakra-checkbox__control': { bg: 'white' } }}>
+          面试限制：
+        </Checkbox>
+        在
+        <DatePicker 
+          selected={date}
+          onChange={date => setDate(date || oneMonthDate)}
+          minDate={new Date()}
+          maxDate={threeMonthsDate}
+          disabled={limit === undefined}
+          customInput={
+            <Input 
+              size="xs"
+              maxW="90px" 
+              marginLeft="2px"
+              marginRight="2px"
+              bgColor="white" 
+              borderRadius="5px"
+              textAlign="center" />}   
+        />
+        之前，我还可以参与
+        <NumberInput 
+          defaultValue={0} 
+          min={0} 
+          max={10} 
+          isDisabled={limit === undefined}
+          marginLeft="2px"
+          marginRight="2px"
+          size="xs"
+          borderRadius="5px"
+          maxW="55px" 
+          bgColor="white">
```

Comment:
similar to another comment, can you move this background style for NumberInput to src/theme?